### DISCUSSION TITLE
17. What else can we do?

### DISCUSSION THREAD

**Discussion by MichaelRimler** [phuse-org/OSTCDA#17] [General]
This is very open ended and initially intended as a catch-all for the future considerations not already covered by the other discussions.

**Comment by manciniedoardo** [comment]
One thought I've had that doesn't really fit in the other categories is the principle of _Conscious Contribution_. That is, open-source code and solutions shouldn't be depend on keen community members working in their spare time as a side project. Rather, sponsor companies can and should consciously _indirectly invest_ in open-source simply by allowing such initiatives to be worked on during standard business hours. Of course, some companies already do this, and all companies need to answer to the requirements of their day-to-day project work, but even still I believe open-source can only truly embed into industry once it is regarded at the same level as project work.
